core/src/main/java/org/apache/iceberg/GenericManifestFile.java

```diff
@@ -245,6 +251,11 @@ public Long deletedRowsCount() {
     return partitions == null ? null : Arrays.asList(partitions);
   }
 
+  @Override
+  public ByteBuffer keyMetadata() {
+    return keyMetadata != null ? ByteBuffer.wrap(keyMetadata) : null;
```

Comment:
I'd rather follow the pattern above 
positive assertion ? positive result : otherwise
```
keyMetadata == null ? null : ByteBuffer.wrap(keyMetadata)
```
Just a personal thing but I always have a hard time with negated boolean checks